raiden/tests/integration/network/transport/test_matrix_transport_assumptions.py

```diff
@@ -61,6 +58,36 @@ def test_assumption_matrix_userid(local_matrix_servers):
         user.get_display_name()
 
     # The userid is valid and the user exists, this should not raise
-    new_client = create_logged_in_client(local_matrix_servers[0])
+    new_client, _ = create_logged_in_client(local_matrix_servers[0])
     user = User(client.api, new_client.user_id)
     user.get_display_name()
+
+
+@pytest.mark.parametrize("matrix_server_count", [2])
+def test_assumption_search_user_directory_returns_federated_users(chain_id, local_matrix_servers):
```

Comment:
Can you add a small docstring what is the assumption that is tested here?

I guess it's `search_user_directory_returns_federated_users` but a bit more detail for those who don't know what that means would be good.